Sources/ValCommand/ValCommand.swift

```diff
@@ -257,7 +272,8 @@ public struct ValCommand: ParsableCommand {
 
   /// Combines the object files located at `objects` into an executable file at `binaryPath`,
   /// logging diagnostics to `log`.
-  private func makeLinuxExecutable<L: Log>(
+  /// Make Val code executable on Windows and Linux(C++ backend)
```

Comment:
```suggestion
  /// Make Val code executable on Windows and Linux.
```
We're not using C++ here.

You should just add "Windows and Linux" at the end of the existing comment. Please don't add a new line:

```swift
/// Combines the object files located at `objects` into a Linux or Windows executable file at `binaryPath`,
/// logging diagnostics to `log`.
```